lib/smart_app_launch/smart_access_brands_validate_endpoint_urls_test.rb

```diff
@@ -7,6 +7,9 @@ class SMARTAccessBrandsValidateEndpointURLs < Inferno::Test
       and available.
     )
 
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@396',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@417'
```

Comment:
I would remove both of these - the test skips if nothing present, but these are really tested elsewhere.

Reply:
Removed 396 since it was still there. Addressed in [0e5baf4](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/0e5baf465ff1c5fe99fce41f42e1529ecf6fbd45)